fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -543,6 +540,7 @@ private Query getFilterQuery(Set<ApiFilter> filters, int perPage) {
     private Pagination<DimensionRow> getResultsPage(Query query, PaginationParameters paginationParameters)
             throws PageNotFoundException {
         int perPage = paginationParameters.getPerPage();
+        validatePerPage(perPage);
```

Comment:
If we're adding it here, we should remove it from `findAllDimensionRowsPaged` as well, so we don't validate twice 😉